Configure

```diff
@@ -359,6 +359,7 @@ my @disablables = (
     "egd",
     "engine",
     "err",
+    "ess",
```

Comment:
You may want to try a build with `no-ess`...  I believe you need a few more `OPENSSL_NO_ESS` guards...

Reply:
Alternatively, just don't make it disablable...